stencil.config.docs.ts

```diff
@@ -23,12 +23,12 @@ export const config: Config = {
             copy: [
                 { src: 'style/color-palette-extended.css' },
                 {
-                    src: '../node_modules/@lundalogik/lime-icons8/assets/',
-                    dest: 'assets/',
+                    src: '../node_modules/@lundalogik/lime-icons8',
+                    dest: 'node_modules/@lundalogik/lime-icons8',
```

Comment:
I guess we're doing this linking for "production" builds of the docs too? If so, it will probably not work on AWS. But we want to move back to GitHub Pages anyway, now that we have Kompendium. So I guess we'll have to do that at the same time…

Reply:
We are still copying all the files for the docs build, it's just copied to another location now so that we have the same structure as the dev build

Reply:
Ah, ok 👍